File: POO/CS/CarStore/CarStore/CarStore/Business/Engine.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CarStore.Business
{
    class Engine
    {
        public Engine()
        {
            // Standard Constructor
        }
        public Engine(TypesOfEngine Type, int Potency)
        {
            this.Type = Type;
            this.Potency = Potency;
        }
        //
        public enum TypesOfEngine { V12, V8, V6, V4 };
        public TypesOfEngine Type { get; set; }
        public int Potency { get; set; }
    }
}

File: POO/J/CarStore/src/business/Engine.java
package business;
public class Engine {
	public Engine() {
		// Standard Constructor
	}
	public Engine(TypesOfEngine type, int potency) {
		super();
		this.type = type;
		this.potency = potency;
	}
	//
	public enum TypesOfEngine { V12, V8, V6, V4 };
	public TypesOfEngine type;
	public int potency;
}
